src/components/RouteDetails.js

```diff
@@ -0,0 +1,13 @@
+import { Grid } from '@mui/material';
+import React from 'react';
+import RouteLine from './RouteLine';
+
+const RouteDetails = ({ routeDetails }) => (
+  <Grid>
+    {routeDetails.map((routeDetail) => (
+      <RouteLine routeDetail={routeDetail} />
```

Comment:
component name should be RouteDetails instead of RouteLine

Reply:
changed to RouteDetail, RouteDetails is already present
From RouteDetails we are passing every route to RouteDetail(RouteLine)